docs/getting-started/tutorial.md

```diff
@@ -0,0 +1,382 @@
+
+If you haven't already followed these [installation instructions](creating-a-new-application-with-the-us-forms-system-starter-app.md), complete those steps first and then come back here!
+
+### Step 2: Getting familiar with the form config
+
+In the starter app code, you'll notice a file under `js/config/` called `form.js`. This is the main file you'll be interacting with to build your form. Almost everything that goes into your form is entered in this file: the questions you want your form to contain, how those questions are displayed to the user, the organization of the different sections of your form, etc. The library code reads through your form config in order to determine how to build your form.
+
+You'll notice several properties in the top level of the `formConfig` object. For now, you don't need to know too much about those. You can read more about them later in our [quick start example formjs file](../building-a-form/quick-start-example-formjs-file.md).
+
+### Step 3: Understanding chapters and pages
+
+The main part of the `formConfig` object that we'll be working with in this tutorial is section within `chapters`. Forms are organized into chapters, which are broader topics, and pages, which are specific sections or pages of questions within that topic. For example, you may have a chapter on personal information, and separate pages within that chapter for name, address, and other contact information.
+
+The starter app sets up some initial chapters and pages for you. There are 2 chapters to this form, `firstChapter` and `secondChapter`. Within `firstChapter`, you'll see the `title` of the chapter and an object containing the `pages` within that chapter. Each page is also an object.
+
+```js
+const formConfig = {
+  title: 'Form',
+  subTitle: 'Test',
+  ...
+  chapters: {
+    firstChapter: {
+      title: 'First Chapter',
+      pages: {
+        firstPage: {
+          path: 'first-chapter/first-page',
+          title: 'First Page',
+          ...
+        }
+      }
+    }
+  }
+}      
+```
+
+Within each page you'll also see `path` and `title` properties: the `path` property takes the url of that page, and the `title`, unsurprisingly, takes the title of that page.
+
+### Step 4: Adding the `schema` object
+
+Now that we have the page structure of our form, we want to start adding questions! We do so by adding the `schema` object to our `firstPage` object.
+
+The `schema` object takes the JSON Schema specification of our questions. You can think about the information contained in the `schema` object as a description of the type of data each question accepts. The `schema` object follows the JSON Schema standard, which describes the allowed shape of JSON objects. For more information about the JSON Schema Standard, see [Understanding JSON Schema](https://spacetelescope.github.io/understanding-json-schema/).
+
+Your config with the `schema` object, which is empty for now, added within `firstPage` should now look like this:
```

Comment:
I'd leave out this first code block because the next one is not that much more complex and is complete/valid syntax whereas this one is not (it's missing the type and properties).

Reply:
Yep, good point!